config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/EntityTypeRegistry.java

```diff
@@ -29,6 +33,15 @@
 @Singleton
 public class EntityTypeRegistry {
 
+    public static final Set<String> NON_ENV_ENTITY_TYPES;
+
+    static {
+        NON_ENV_ENTITY_TYPES = new HashSet<>();
+        NON_ENV_ENTITY_TYPES.add(EntityTypes.FOLDER_TYPE);
+        NON_ENV_ENTITY_TYPES.add(EntityTypes.POLICY_TYPE);
+        NON_ENV_ENTITY_TYPES.add(EntityTypes.SERVICE_TYPE);
+        NON_ENV_ENTITY_TYPES.add(EntityTypes.ENCAPSULATED_ASSERTION_TYPE);
+    }
```

Comment:
We have the same code in `BuilderConstants`, why can't we have it at 1 common place?

Reply:
Done